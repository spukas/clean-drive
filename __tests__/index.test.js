/* eslint-env jest */

import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

const App = () => (
  <div>
    <style jsx>{`
      p {
        color: red;
      }
    `}
    </style>
    <p>Hello World!</p>
  </div>
);

describe('With Enzyme', () => {
  it('App shows "Hello world!"', () => {
    const app = shallow(<App />);

    expect(app.find('p').text()).toEqual('Hello World!');
  });
});

describe('With Snapshot Testing', () => {
  it('App shows "Hello world!"', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
