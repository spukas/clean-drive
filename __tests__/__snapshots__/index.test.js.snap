// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<div
  className="jsx-316859968"
>
  <p
    className="jsx-316859968"
  >
    Hello World!
  </p>
</div>
`;
